logicrunner/handle_calls.go

```diff
@@ -218,12 +215,11 @@ func (h *HandleCall) handleActual(
 	}
 
 	done, err := h.dep.WriteAccessor.Begin(ctx, flow.Pulse(ctx))
-	defer done()
-
 	if err != nil {
```

Comment:
Might be another error (not currently, but in future). It will be hard to debug.